integration/specs/Modal/modal-11.spec.js

```diff
@@ -1,14 +1,28 @@
 const PageModal = require('../../../src/components/Modal/pageObject');
 const PageLookup = require('../../../src/components/Lookup/pageObject');
+const PageDatePicker = require('../../../src/components/DatePicker/pageObject');
 const { ESCAPE_KEY, ENTER_KEY, ARROW_DOWN_KEY } = require('../../constants');
 
 const BUTTON = '#button-11';
 const MODAL = '#modal-11';
+const MODAL_DATEPICKER = '#modal-datepicker-11';
 const TITLE_INPUT = '#modal-11 input[placeholder="Enter company name"]';
 const DATE_PICKER_INPUT = '#modal-11 input[placeholder="Select a date"]';
 const TIME_PICKER_INPUT = '#modal-11 input[name="time"]';
 const MODAL_LOOKUP = '#modal-lookup-11';
 
+const getScrollTop = () => {
```

Comment:
If the intent of this is return the scroll top position then let's call it `getScrollTopPosition`